api/graphql/types/recurring_reservation/serializers.py

```diff
@@ -437,3 +343,276 @@ def save(self, **kwargs: Any) -> RecurringReservation:
+        if next_reservation is None:
+            next_reservation = instance.reservations.filter(begin__lt=now).going_to_occur().order_by("-begin").first()
+        if next_reservation is None:
+            next_reservation = instance.reservations.filter(begin__gte=now).order_by("begin").first()
+        if next_reservation is None:
+            next_reservation = instance.reservations.filter(begin__lt=now).order_by("-begin").first()
+
+        return ReservationDetails(
+            name=next_reservation.name,
+            description=next_reservation.description,
+            num_persons=next_reservation.num_persons,
+            # If we needed to use a cancelled or denied reservation,
+            # state for the new reservation should be CONFIRMED instead.
+            state=(
+                next_reservation.state
+                if next_reservation.state in ReservationStateChoice.states_going_to_occur
+                else ReservationStateChoice.CONFIRMED
+            ),
+            type=next_reservation.type,
+            working_memo=next_reservation.working_memo,
+            #
+            handled_at=next_reservation.handled_at,
+            confirmed_at=next_reservation.confirmed_at,
+            #
+            applying_for_free_of_charge=next_reservation.applying_for_free_of_charge,
+            free_of_charge_reason=next_reservation.free_of_charge_reason,
+            #
+            reservee_id=next_reservation.reservee_id,
+            reservee_first_name=next_reservation.reservee_first_name,
+            reservee_last_name=next_reservation.reservee_last_name,
+            reservee_email=next_reservation.reservee_email,
+            reservee_phone=next_reservation.reservee_phone,
+            reservee_organisation_name=next_reservation.reservee_organisation_name,
+            reservee_address_street=next_reservation.reservee_address_street,
+            reservee_address_city=next_reservation.reservee_address_city,
+            reservee_address_zip=next_reservation.reservee_address_zip,
+            reservee_is_unregistered_association=next_reservation.reservee_is_unregistered_association,
+            reservee_language=next_reservation.reservee_language,
+            reservee_type=next_reservation.reservee_type,
+            #
+            billing_first_name=next_reservation.billing_first_name,
+            billing_last_name=next_reservation.billing_last_name,
+            billing_email=next_reservation.billing_email,
+            billing_phone=next_reservation.billing_phone,
+            billing_address_street=next_reservation.billing_address_street,
+            billing_address_city=next_reservation.billing_address_city,
+            billing_address_zip=next_reservation.billing_address_zip,
+            #
+            user=next_reservation.user,
+            purpose=next_reservation.purpose,
+            home_city=next_reservation.home_city,
```

Comment:
`age_group` ?

Reply:
Age group is added in `bulk_create_reservation_for_periods` from the recurring reservation, so it doesn't need to be here.